src/lib/modules/placeBlock.ts

```diff
@@ -88,14 +88,15 @@ module.exports.server = (serv, { version }) => {
   }
 }
 
-module.exports.player = function (player, serv, { version }) {
+export const player = function (player: Player, serv: Server, { version }: Options) {
   const registry = require('prismarine-registry')(version)
   const blocks = registry.blocks
 
   player._client.on('block_place', async ({ direction, location, cursorY } = {}) => {
     const referencePosition = new Vec3(location.x, location.y, location.z)
     const block = await player.world.getBlock(referencePosition)
     block.position = referencePosition
+    //@ts-ignore TODO
     block.direction = direction
```

Comment:
Not sure why this pattern is widely used across prismarine projects, imo it shouldn't as it makes debugging hard